pydiff.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
```

Comment:
I don't LOVE adding this file and digging ourselves even deeper into `lit`, but I'm OKAY with it for now. Once we have project files we should definitely tackle #35 though.

Reply:
If we make nightly builds right after 3.12 that have 4.0 defaults, then we can just list a bunch of specific builds to test against. I have tested them locally on my machine, running lit with a 4.0 build.